docs/design/chart_format.md

```diff
@@ -394,7 +370,7 @@ The following URI schemes must be supported:
 
 
 #### Special Form Google Storage and S3
-The [Google Storage scheme](https://cloud.google.com/storage/docs/interoperability) (`gs://`)  and the S3 scheme (`s3://`) can also be support. These forms do not follow the URL standard (bucket name is placed in the host field), but their behavior is well-documented.
+The [Google Storage scheme](https://cloud.google.com/storage/docs/interoperability) (`gs://`)  and the S3 scheme (`s3://`) can also be supported. These forms do not follow the URL standard (bucket name is placed in the host field), but their behavior is well-documented.
 
 
 Examples:
```

Comment:
In the Short Form section below, it should be noted that when a short form URL is used for an object storage service URL (e.g., starting with `gs` or `s3`), then the object storage service scheme appears as the `DOMAIN` segment of the reference (e.g., `helm:gs/kubernetes-charts/nginx-1.2.3.tgz`).